src/UI/templates/js/Dropzone/File/dropzone.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * Provides the behavior of all dropzone types.
+ *
+ * @author nmaerchy <nm@studer-raimann.ch>
+ */
+
+var il = il || {};
+il.UI = il.UI || {};
+(function ($, UI) {
+    UI.dropzone = (function ($) {
+
+        /**
+         * Contains all css classes used for dropzone manipulation.
+         * These classes MUST NOT have the . symbol at the beginning.
+         */
+        var CSS = {
+            "darkenedBackground": "modal-backdrop in", // <- bootstrap classes, should not be changed
+            "darkenedDropzoneHighlight": "darkened-highlight",
+            "defaultDropzoneHighlight": "default-highlight",
+            "dropzoneDragHover": "drag-hover"
+        };
+
+        /**
+         * Contains all css selectors used for dropzone manipulation.
+         * Selectors MUST be declared like in css.
+         * e.g.
+         *  .this-is-a-class
+         *  #this-is-a-id
+         */
+        var SELECTOR = {
+            "darkenedBackground": "#il-dropzone-darkened",
+            "dropzones": ".il-dropzone"
+        };
+
+        /**
+         * Contains all supported dropzone types.
+         * The type MUST be equal to the full qualified class name used in php.
+         * NOTE backslashes needs to be removed.
+         * e.g. ILIAS\UI\Component\Dropzone\Standard -> Standard
+         */
+        var DROPZONE = {
+            "standard": "Standard",
+            "wrapper": "Wrapper"
+        };
+
+        var _darkenedBackground = false;
```

Comment:
This is just a "convention" which we could append to our JS Readme. But I's like to do this in a separate PR